packages/calcite-components/src/components/stepper-item/assets/stepper-item/t9n/messages.json

```diff
@@ -1,5 +1,3 @@
 {
-  "complete": "Completed step",
-  "previousStep": "Previous step",
```

Comment:
Message removal treads into a territory of technically being breaking, but probably not affecting many, so we have a couple of options:

1. Restore these strings and remove them in a separate one (similar to https://github.com/Esri/calcite-design-system/pull/8099) so that the responsive feature isn't seen as itself being breaking in the changelog
2. Mark this PR as breaking and be clear about it only affecting anyone actually using the strings object or its type

I'm leaning more towards 1.

Reply:
Added https://github.com/Esri/calcite-design-system/pull/8222